src/test/java/org/mule/tools/devkit/sonar/checks/maven/DevkitLatestVersionCheckTest.java

```diff
@@ -0,0 +1,55 @@
+package org.mule.tools.devkit.sonar.checks.maven;
+
+
+import com.google.common.collect.Iterables;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.junit.Test;
+import org.mule.tools.devkit.sonar.checks.ConnectorIssue;
+import org.mule.tools.devkit.sonar.utils.PomUtils;
+
+import javax.xml.stream.XMLStreamException;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class DevkitLatestVersionCheckTest {
+
+    @Test
+    public void checkDevkitVersionIsTrue() throws IOException, XmlPullParserException, XMLStreamException {
```

Comment:
Provide a more specific name to what is being tested such as `checkDevkitVersionIsLatest()`